app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/ytdliability/upload_mandatory_evidence.scala.html

```diff
@@ -0,0 +1,70 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.config.ViewConfig
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.returns.yeartodatelliability.routes
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.actions.RequestWithSessionData
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.DisposalDate
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.views.components.AutoCompleteType
+
+@this(
+        mainTemplate: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.main_template,
+        pageHeading: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.page_heading,
+        submitButton: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.submit_button,
+        errorSummary: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.error_summary,
+        formWrapper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF,
+        backLinkComponent: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.back_link,
+        returnToSummaryLink: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.return_to_summary_link,
+        textInput: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.text_input,
+        caption: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.caption,
+        details: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.details
+)
+
+@(form: Form[String], backLink: Call)(implicit request: RequestWithSessionData[_], messages:Messages, appConfig: ViewConfig)
+@key = @{ "mandatoryEvidence" }
+
+@title = @{messages(s"$key.title")}
+@hasErrors = @{form.hasErrors || form.hasGlobalErrors}
+
+@mainTemplate(title = title, userType = request.userType, hasErrors = hasErrors, withSignOutLink = false) {
+
+    @backLinkComponent(backLink)
+
+    @if(hasErrors) {
+        @errorSummary(form)
+    }
+
+    @caption(messages("ytdLiability.caption"))
+
+    @formWrapper(routes.YearToDateLiabilityFirstReturnController.uploadMandatoryEvidenceSubmit(), 'novalidate -> "novalidate") {
+        @textInput(
```

Comment:
this is a page that will behave the actual file upload stuff in it eventually - I made it a simple text input page for now just so that we have something.